src/util/jszip.js

```diff
@@ -1,9155 +1,9412 @@
 /*!
```

Comment:
Tracing back my steps, I didn't leave the updated version of jszip, it was different because I left it unminified. Now it's exactly as it is currently in develop.